package.json

```diff
@@ -63,6 +67,7 @@
     "run": "web-ext run -s dist --firefox=nightly",
     "clean": "rm -rf dist && shx mkdir -p dist/popup",
     "watch": "npm-run-all --parallel watch:*",
+    "copy:locales": "node pontoon-to-webext",
```

Comment:
Probably worth adding a `watch:locales` around here, too, so that this `copy:locales` gets run whenever the locales are updated.